runtime/tr.source/trj9/build/toolcfg/gnu/common.mk

```diff
@@ -114,13 +114,15 @@ CX_OPTFLAG?=$(CX_DEFAULTOPT)
 CX_FLAGS_PROD+=$(CX_OPTFLAG)
 
 ifeq ($(HOST_ARCH),x)
+    CX_FLAGS+=-mfpmath=sse -msse -msse2 -fno-strict-aliasing -fno-math-errno -fno-rounding-math -fno-trapping-math -fno-signaling-nans
```

Comment:
I think a comment near hear about the flags and what they change is warranted for reference purposes.